app/src/main/java/com/example/warehousemanagement/domain/model/ExportPackages.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.warehousemanagement.domain.model
+
+import java.util.Date
+
+data class ExportPackages(
```

Comment:
Same issue with ImportP, please modiy it nha

Reply:
Done

Reply:
add List product